docs/samples/bookinfo.md

```diff
@@ -170,97 +132,97 @@ route requests to all available versions of a service in a random fashion.
 
 1. Set the default version for all microservices to v1.
 
-   ```bash
-   $ istioctl create -f route-rule-all-v1.yaml
-   ```
-
-   You can display the routes that are defined with the following command:
-
-   ```bash
-   $ istioctl get route-rules -o yaml
-   kind: route-rule
-   name: ratings-default
-   namespace: default
-   spec:
-     destination: ratings.default.svc.cluster.local
-     precedence: 1
-     route:
-     - tags:
-         version: v1
-       weight: 100
-   ---
-   kind: route-rule
-   name: reviews-default
-   namespace: default
-   spec:
-     destination: reviews.default.svc.cluster.local
-     precedence: 1
-     route:
-     - tags:
-         version: v1
-       weight: 100
-   ---
-   kind: route-rule
-   name: details-default
-   namespace: default
-   spec:
-     destination: details.default.svc.cluster.local
-     precedence: 1
-     route:
-     - tags:
-         version: v1
-       weight: 100
-   ---
-   kind: route-rule
-   name: productpage-default
-   namespace: default
-   spec:
-     destination: productpage.default.svc.cluster.local
-     precedence: 1
-     route:
-     - tags:
-         version: v1
-       weight: 100
-   ---
-   ```
-
-   > Note: In the current Kubernetes implemention of Istio, the rules are stored in ThirdPartyResources.
-   > You can look directly at the stored rules in Kubernetes using the `kubectl` command. For example,
-   > the following command will display all defined rules:
-   > ```bash
-   > $ kubectl get istioconfig -o yaml
-   > ```
+```bash
+istioctl create -f \
+    https://raw.githubusercontent.com/istio/istio/master/demos/apps/bookinfo/route-rule-all-v1.yaml
```

Comment:
Once again as i noted earlier, lets not download from URLs. The delivery medium will change. Its better to have one download pointer at the top of the tutorial (download from here or there) and then reference all files locally.